src/screens/block/block.vue

```diff
@@ -132,16 +96,26 @@ export default {
   ],
   components: { AePanel, RelativeTime, Transaction, Loader },
   mixins: [currentTime],
-  computed: mapState('blocks', [
-    'height',
-    'generation'
-  ]),
+  computed: {
+    ...mapState('blocks', {
+      block: ({ block }) => block,
+      height: ({ height }) => height
+    }),
```

Comment:
Try to use `mapState('blocks', ['block', 'height'])` instead

Reply:
This might be worth a read in addition to Denis comment: https://vuex.vuejs.org/guide/state.html